### ISSUE TITLE
Made some notes for the final exam feel free to use if you want :)💡

### ISSUE THREAD

**Issue by yucheng11122017** [nus-cs2103-AY2122S1/forum#418]
This is just like a glossary because I found there was too many terms in general...
[Terms.docx](https://github.com/nus-cs2103-AY2122S1/forum/files/7551534/Terms.docx)
Notes on UML diagrams 
[UML_diagrams.docx](https://github.com/nus-cs2103-AY2122S1/forum/files/7551535/UML_diagrams.docx)
This is just a copy of all the toc of all the weeks. Made it to ctrl f to find the topics that I need info on and go to that week instead of having to click through all the weeks 
[Summary_Topics_Week.docx](https://github.com/nus-cs2103-AY2122S1/forum/files/7551536/Summary_Topics_Week.docx)

Definitely not perfect/high quality and may have some mistakes. Feel free to point them out here! :)
Also reminder that the final exams only PDF is allowed so remember to change them to pdf before!

**Comment by SkyBlaise99** [comment]
Wait only pdf is allowed?

**Comment by yucheng11122017** [comment]
@SkyBlaise99 yup it was stated in the final exam info

**Comment by SkyBlaise99** [comment]
Then I guess u could just post the pdf version lol, altho I can go turn it into pdf myself

Here are the pdf versions:

* [Terms.pdf](https://github.com/nus-cs2103-AY2122S1/forum/files/7554197/Terms.pdf)
* [UML_diagrams.pdf](https://github.com/nus-cs2103-AY2122S1/forum/files/7554198/UML_diagrams.pdf)
* [Summary_Topics_Week.pdf](https://github.com/nus-cs2103-AY2122S1/forum/files/7554194/Summary_Topics_Week.pdf)

**Comment by yucheng11122017** [comment]
@SkyBlaise99 yeah but i figured some people might want to add some stuff before converting it

**Comment by SkyBlaise99** [comment]
True, anyways I left the pdf versions there if they dun wanna update anymore.

**Comment by Chen-Yiyang** [comment]
Thank you!
